app/src/main/java/com/hedvig/app/feature/embark/ui/MoreOptionsActivity.kt

```diff
@@ -3,29 +3,19 @@ package com.hedvig.app.feature.embark.ui
 import android.content.Context
 import android.content.Intent
 import android.os.Bundle
-import androidx.core.view.isVisible
 import androidx.core.view.updatePaddingRelative
 import com.hedvig.app.BaseActivity
 import com.hedvig.app.R
 import com.hedvig.app.databinding.ActivityMoreOptionsBinding
 import com.hedvig.app.feature.onboarding.MoreOptionsViewModel
-import com.hedvig.app.feature.settings.MarketManager
-import com.hedvig.app.util.extensions.view.setHapticClickListener
-import com.hedvig.app.util.extensions.view.updateMargin
 import com.hedvig.app.util.extensions.viewBinding
 import dev.chrisbanes.insetter.doOnApplyWindowInsets
 import dev.chrisbanes.insetter.setEdgeToEdgeSystemUiFlags
-import org.koin.android.ext.android.inject
 import org.koin.android.viewmodel.ext.android.viewModel
 
 class MoreOptionsActivity : BaseActivity(R.layout.activity_more_options) {
```

Comment:
Where is this screen used now? 🤔

Reply:
Shown when pressing "App information" in the overflow menu!